src/FocusTrap.ts

```diff
@@ -54,13 +55,25 @@ export const FocusTrap = defineComponent({
 
   setup(props, { slots, emit }) {
     let trap: FocusTrapI | null
-    const el = ref<HTMLElement | null>(null)
+    const el = ref<HTMLElement | ComponentPublicInstance | null>(null)
```

Comment:
Let's actually change this to

```suggestion
    const el = ref<HTMLElement | ComponentPublicInstance | undefined>()
```